doc/kuksa_analysis.md

```diff
@@ -11,14 +11,6 @@ The extended list of design topics for Kuksa Databroker is available in the [Kuk
 ### 3. Use Cases
 The use cases for Kuksa Databroker is available in the [Kuksa use cases](./kuksa_analysis/kuksa_use_cases.md)
 
-### 4. New API Definition
```

Comment:
Why should this section not be part of the analysis? I placed it here because I thought it would be good to track continuous changes, improvements, and proposed additions for further development of the API, along with their release versions. For example:

0.5.0 -> Initial release of kuksa.val.v2
0.6.0 -> Some new feature
0.6.1 -> Bug fixes

Where would be an appropriate place to include these?

Reply:
I think you are right moving protocol things all together, fine for me.
What I proposed can be seen in the release history also.